classes/log/store.php

```diff
@@ -65,10 +65,14 @@ class store extends php_obj implements log_writer {
      * @param log_manager $manager
      */
     public function __construct(log_manager $manager) {
+        global $CFG;
         $this->helper_setup($manager);
         $this->logguests = $this->get_config('logguests', 1);
         $routes = $this->get_config('routes', '');
         $this->routes = $routes === '' ? [] : explode(',', $routes);
+        if (!empty($CFG->debug) and $CFG->debug >= DEBUG_DEVELOPER) {
```

Comment:
I might have used `&&` instead of `and` to protect against future changes, since some people don't understand there's a slight difference in precedence between the two. Not too worried about that though, just something to be aware of 😄

Reply:
I Agree, just used an old piece of code from another plugin I have.
Better fix it, at some point.